testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -528,6 +507,93 @@ QUnit.module('datebox tests', moduleConfig, () => {
     });
 });
 
+QUnit.module('toolbar buttons', {}, () => {
+    const types = [ 'date', 'datetime' ];
+
+    const buttons = [
+        {
+            optionName: 'todayButtonText',
+            name: 'Today',
+            newText: 'newTodayText',
+            selector: TODAY_BUTTON_SELECTOR,
+            localizationMessageKey: 'dxCalendar-todayButtonText',
+        },
+        {
+            optionName: 'applyButtonText',
+            name: 'Done',
+            newText: 'newDoneText',
+            selector: APPLY_BUTTON_SELECTOR,
+            localizationMessageKey: 'OK',
+        },
+        {
+            optionName: 'cancelButtonText',
+            name: 'Cancel',
+            newText: 'newCancelText',
+            selector: CANCEL_BUTTON_SELECTOR,
+            localizationMessageKey: 'Cancel',
+        },
+    ];
+
+    types.forEach(type => {
+        buttons.forEach(button => {
```

Comment:
let's move all tests to the single `buttons.forEach`

Reply:
https://github.com/DevExpress/DevExtreme/pull/22991/commits/282428fc00149b70ee170726a3ac4ad578a751d3